scitasexternal/packages/amber/package.py

```diff
@@ -35,33 +34,43 @@ class Amber(Package):
     """
 
     homepage = 'http://ambermd.org'
-    url = 'file://fake/amber16.tgz'
+    url = 'file:///fake/path/to/amber-16.tgz'
 
+    version('18', 'ae5438d3f1e2d3379d5664c34193ae48')
     version('16', '652e24512146e966a0a50335572ebd66')
 
     # FIXME: separate ambertools and amber
 
     variant('mpi', default=True, description='Enables MPI support')
     variant('openmp', default=False, description='Enables OpenMP support')
     variant('cuda', default=False, description='Enables CUDA support')
-    # variant('X', default=False, description='Enables X11 support')
-
-    depends_on('mpi', when='+mpi')
+    variant('X', default=False, description='Enables X11 support')
+    # As of September 2018 AmberTools packmol_memgen is the only
+    # boost-dependent part. However, AmberTools fails to compile if boost
+    # support is included, due to a bug. By putting packmol as a variant,
+    # AmberTools can be compiled in spite of this specific issue.
+    variant('packmol', default=False,
+            description='Compiles the optional packmol_memgen')
+
+    depends_on('boost', when='+packmol')
```

Comment:
It seems the packmol variant is only used to remove the boost dependency, but this will not work if there is a system boost installed.

Can we explicitly disable packmol if possible?

Reply:
My latest commit removed this variant, because the bug has been corrected in the meantime.

In any case, explicit disabling of this component was not possible.